### PR TITLE
Fix #125: remove redundant convolution doc. 

### LINKED ISSUE #110: Sample format: what do +/-Infinity and NaN mean?
> Originally reported on W3C Bugzilla [ISSUE-17394](https://www.w3.org/Bugs/Public/show_bug.cgi?id=17394) Tue, 05 Jun 2012 12:14:04 GMT
> Reported by Michael[tm] Smith
> Assigned to 

Audio-ISSUE-19 (SamplingFormat): Define sampling format [Web Audio API]

http://www.w3.org/2011/audio/track/issues/19

Raised by: Marcus Geelnard
On product: Web Audio API

In the internal sample format the interpretation of Float32 NaN, +-Inf etc is not defined. More generally, it seems to not be defined what the sampling format of everything is (AudioBuffer, WaveShaperNode, etc).

### LINKED ISSUE #125: (LinearConvolution): Linear convolution effect is defined by Wikipedia
> Originally reported on W3C Bugzilla [ISSUE-17403](https://www.w3.org/Bugs/Public/show_bug.cgi?id=17403) Tue, 05 Jun 2012 12:28:55 GMT
> Reported by Michael[tm] Smith
> Assigned to 

Audio-ISSUE-69 (LinearConvolution): Linear convolution effect is defined by Wikipedia [Web Audio API]

http://www.w3.org/2011/audio/track/issues/69

Raised by: Philip Jägenstedt
On product: Web Audio API

https://dvcs.w3.org/hg/audio/raw-file/tip/webaudio/specification.html#ConvolverNode

links to

https://dvcs.w3.org/hg/audio/raw-file/tip/webaudio/specification.html#Convolution-section

links to

http://en.wikipedia.org/wiki/Convolution

The definition should be in the specification itself. A problem that is not addressed at all is how phase/latency depends on the length of the convolution buffer. In other words, it must be defined exactly how each sample value is calculated from the the input signal and the impulse response buffer.

### PR DESCRIPTION
**Pull request by joeberkovitz**
Moved normative section on ConvolverNode channel configurations into ConvolverNode section where it belongs.

### CODE REVIEW

**Comment by joeberkovitz** [comment]
Commit comment is not correct: this fixes #125

**Comment by rtoy** [comment]
Its' a bit hard to follow exactly what changed except that you did remove some old stuff on the motivation and background which doesn't really belong in the spec.

lgtm.
